scripts/visual-baselines.js

```diff
@@ -0,0 +1,71 @@
+/* jshint node: true */
+'use strict';
+
+const fs = require('fs-extra');
+const path = require('path');
+const rimraf = require('rimraf');
+const logger = require('@blackbaud/skyux-logger');
+
+const {
+  exec,
+  dirHasChanges
+} = require('./utils');
+
+const baselineScreenshotsDir = 'screenshots-baseline';
+const tempDir = '.skypagesvisualbaselinetemp';
+
+function handleBaselineScreenshots() {
+  const buildId = new Date().getTime();
+  const branch = 'master';
+  const opts = { cwd: tempDir };
+  const gitUrl = process.env.VISUAL_BASELINES_REPO_URL;
+
+  return Promise.resolve()
+    .then(() => exec('git', ['config', '--global', 'user.email', '"sky-build-user@blackbaud.com"']))
+    .then(() => exec('git', ['config', '--global', 'user.name', '"Blackbaud Sky Build User"']))
+    .then(() => exec('git', ['clone', gitUrl, '--single-branch', tempDir]))
+    .then(() => fs.copy(
+      baselineScreenshotsDir,
+      path.resolve(tempDir, baselineScreenshotsDir)
+    ))
+    .then(() => exec('git', ['checkout', branch], opts))
+    .then(() => exec('git', ['status'], opts))
+    .then(() => exec('git', ['add', baselineScreenshotsDir], opts))
+    .then(() => exec('git', ['commit', '-m', `Build #${buildId}: Added new baseline screenshots. [ci skip]`], opts))
+    .then(() => exec('git', ['push', '-fq', 'origin', branch], opts))
```

Comment:
Is it necessary to force push?

Reply:
This is how we're currently handling it for skyux2. https://github.com/blackbaud/skyux2/blob/master/scripts/visual-baseline.sh#L31

I'm thinking that it is necessary in case the master branch has any special "blockers" on pushing to the master branch.